test/appium/views/home_view.py

```diff
@@ -257,7 +257,7 @@ def get_chat(self, username, community=False, wait_time=30):
         self.driver.info("Looking for chat: '%s'" % username)
         chat_element = ChatElement(self.driver, username[:25], community=community)
         if not chat_element.is_element_displayed(10):
-            self.notifications_unread_badge.wait_and_click(wait_time)
+            self.open-activity-center-button.wait_and_click(wait_time)
```

Comment:
we can't really use it here, as it is always visible. we need accessibility id on the badge itself, as we rely on behavior whether it is visible or not